.github/ISSUE_TEMPLATE/descri.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Descri
```

Comment:
I think this will be used as the heading of the template. So description would be better.

Reply:
Do you really need this and the bug report?

Reply:
No, this should be deleted. I'm really too stupid for the GUI template generator on the GitHub page.